Eurovision Song Contest Analysis/Model/README.md

```diff
@@ -0,0 +1,45 @@
+
+
+**Eurovision Song Contest Anlaysis**

```

Comment:
Use the `<h1>` tag for the project name.

Reply:
Done.